paddle/pybind/pybind.cc

```diff
@@ -40,6 +42,23 @@ static size_t UniqueIntegerGenerator() {
   return generator.fetch_add(1);
 }
 
+std::once_flag gflags_init_flag;
+
+void InitGflags(std::vector<std::string> argv_vec) {
```

Comment:
argv_vec => argv.  The `v` in argv has been "vectors".

Reply:
Done